core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -29,6 +29,7 @@
 import hudson.model.Api;
 import hudson.model.ModelObject;
 import jenkins.YesNoMaybe;
+import jenkins.model.Configuration;
```

Comment:
That doesn't seem to be a great choice for configuration options outside `Jenkins` itself.